lib/streamlit/__init__.py

```diff
@@ -209,3 +209,7 @@ def _update_logger() -> None:
 experimental_rerun = _rerun
 experimental_data_editor = _main.experimental_data_editor
 experimental_connection = _connection_factory
+
+import streamlit.column_config as _column_config
```

Comment:
nit: This should be at the top of imports?

Reply:
yep, done 👍